### PR TITLE
Clientbackup script

### PR DESCRIPTION
**Pull request by tcsimerson**
ClientBackup.ps1 is a PowerShell script designed to backup a Windows client using a command line specified policy name and API key.
[ClientBackup.zip](https://github.com/VeritasOS/netbackup-api-code-samples/files/5225088/ClientBackup.zip)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +0,0 @@
-### NetBackup API Code Samples
```
Comment (rkennedy) [inline README.md]: Please don't delete the top-level readme.

File: recipes/powershell/ClientBackup.ps1 [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+    * Minimum specific privileges:  
+        * Global -> NetBackup management -> NetBackup images -> View
+        * Global -> Protection -> Policies -> View
+        * Global -> Protection -> Policies -> Manual Backup
+* PowerShell Execution Policy needs to be opened
+#>
+
+
+param (
+    [string]$p = $(throw "Please speicfy the policy name using -p parameter."),
```
Comment (rkennedy) [inline recipes/powershell/ClientBackup.ps1]:
```suggestion
    [string]$p = $(throw "Please specify the policy name using -p parameter."),
```

**Review by rkennedy — CHANGES_REQUESTED** [review]

**Comment by tcsimerson** [comment]
Rob,

Sorry about this.  I’m a GitHub noob.  Not sure how to fix this.  Suggestions?

Tom C. Simerson
Account SE and Data Protection TC
Veritas Technologies LLC

Mobile: (972) 750 6519
tom.simerson@veritas.com<mailto:tom.simerson@veritas.com>

[cid:image001.png@01D69CA3.00F951B0]
Simplifying Complexity

This message (including any attachments) is intended only for the use of the individual or entity to which it is addressed and may contain information that is non-public, proprietary, privileged, confidential, and exempt from disclosure under applicable law or may constitute as attorney work product. If you are not the intended recipient, you are hereby notified that any use, dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, notify us immediately by telephone and (i) destroy this message if a facsimile or (ii) delete this message immediately if this is an electronic communication.

**Comment by rkennedy** [comment]
If you don't already have a directory with your work in it, then clone this repo and check out this branch. (You might already have something, since it's what you used to submit this in the first place.) Then, make the changes I've requested:

1. To restore the deleted readme, I think it's sufficient to run `git checkout origin/master README.md`. When you run `git status`, that file should appear as staged (in green).
2. Fix the spelling, and use `git add` to stage the fixed file, too.

Run `git commit` and describe the changes, and finally run `git push` to send them to GitHub. This pull request should get updated automatically, and I'll get a notification.

If you hit any problems, you can find me on Teams, and I can help out or take over any time.

**Review by rkennedy — APPROVED** [review]

**Pull request merged**